packages/core/lib/client/wallet-rpc.test.js

```diff
@@ -37,6 +37,14 @@ describe('WalletRpc', () => {
     testRpcEndpoint(service, WalletRpc.add);
   });
 
+  it('createAccountDocuments', async () => {
+    testRpcEndpoint(service, WalletRpc.createAccountDocuments);
```

Comment:
what's the expected result of making this call? Is there something you can assert? 

Same goes for other tests in this file.

Reply:
testRpcEndpoint will assert the endpoint 

`
export function testRpcEndpoint(service, rpcMethod) {
  const params = [];
  rpcMethod(...params);
  const endpoint = resolveServiceEndpoint(
    service,
    service.routes[rpcMethod.name],
  );

  expect(getRpcClient().request).toBeCalled();
  expect(getRpcClient().request.mock.calls[0][0]).toBe(endpoint);
}`